lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -120,17 +119,6 @@ class ReportUIFeatures {
     this.onMediaQueryChange(mediaQuery);
   }
 
-  _setupSmoothScroll() {
-    for (const el of this._dom.findAll('a.lh-gauge__wrapper', this._document)) {
-      const anchorElement = /** @type {HTMLAnchorElement} */ (el);
-      anchorElement.addEventListener('click', e => {
-        e.preventDefault();
-        window.history.pushState({}, '', anchorElement.hash);
-        this._dom.find(anchorElement.hash, this._document).scrollIntoView({behavior: 'smooth'});
```

Comment:
wait why?

Reply:
was mentioned in the design meeting. @yuinchien asked if the animation could be shorter. i said what if it were just removed completely. i thought everyone said ok

idea was just - user wants to see a particular part of the report, so just show them as soon as intent is made w/o the fanfare of a scroll animation.

Reply:
yuin and i are still attached to it. i got a branch for fixing the highlighter anim

Reply:
@paulirish whats up with this